src/shared/Core.Tests/WslUtilsTests.cs

```diff
@@ -100,15 +101,15 @@ public void WslUtils_CreateWslProcess()
             string expectedFileName = WslUtils.GetWslPath();
             string expectedArgs = $"--distribution {distribution} --exec {command}";
 
-            Process process = WslUtils.CreateWslProcess(distribution, command);
+            ChildProcess process = WslUtils.CreateWslProcess(distribution, command, Mock.Of<ITrace2>());
 
             Assert.NotNull(process);
-            Assert.Equal(expectedArgs, process.StartInfo.Arguments);
-            Assert.Equal(expectedFileName, process.StartInfo.FileName);
-            Assert.True(process.StartInfo.RedirectStandardInput);
-            Assert.True(process.StartInfo.RedirectStandardOutput);
-            Assert.False(process.StartInfo.RedirectStandardError);
-            Assert.False(process.StartInfo.UseShellExecute);
+            Assert.Equal(expectedArgs, process.Process.StartInfo.Arguments);
+            Assert.Equal(expectedFileName, process.Process.StartInfo.FileName);
+            Assert.True(process.Process.StartInfo.RedirectStandardInput);
+            Assert.True(process.Process.StartInfo.RedirectStandardOutput);
+            Assert.False(process.Process.StartInfo.RedirectStandardError);
+            Assert.False(process.Process.StartInfo.UseShellExecute);
```

Comment:
I wonder if this is another property on `ChildProcess` we could create to avoid churn..

```csharp
public ProcessStartInfo StartInfo => Process.StartInfo;
```